hayhooks/pipelines/search/pipeline_wrapper.py

```diff
@@ -60,18 +42,17 @@ def create_pipeline(self) -> Pipeline:
         pipe.connect("search.documents", "prompt_builder.documents")
         pipe.connect("prompt_builder", "llm")
 
-        return pipe
+        self.pipeline = pipe
 
     def run_api(
         self,
         question: str,
         max_results: int,
-        search_depth: Literal["basic", "advanced"] = "basic",
-        include_domains: Optional[list[str]] = None,
-        exclude_domains: Optional[list[str]] = None,
+        search_depth: str,
```

Comment:
Using complex types just confuses Google as it only takes a subset of OpenAPI 3.0